swagger_django_generator/generator.py

```diff
@@ -79,8 +79,8 @@ def render_to_string(filename, context, path=None):
     :return: The rendered template as a string
     """
     return jinja2.Environment(
-        # loader=jinja2.FileSystemLoader(path or './'),
-        loader=jinja2.PackageLoader("swagger_django_generator", "templates"),
+        loader=jinja2.FileSystemLoader(path or './swagger_django_generator/templates'),
```

Comment:
Can't we keep this as a `PackageLoader`?

Reply:
ah darn forgot to comment this out again...